src/region.js

```diff
@@ -197,8 +188,13 @@ const Region = MarionetteObject.extend({
 
   // Override this method to change how the new view is appended to the `$el` that the
   // region is managing
-  attachHtml(view) {
-    this.el.appendChild(view.el);
+  attachHtml(view, shouldReplace) {
+    if (shouldReplace) {
```

Comment:
`domixin.attachHTML` is a name I am not attached to. If you have a better name, I am open to suggestions.